src/ui.rs

```diff
@@ -105,13 +126,20 @@ pub fn event_action(state: &mut State, event: &Event) -> Action {
                 state.render_infobar = !state.render_infobar;
                 state.process_action(Action::ReRender)
             }
-
-            W | PageUp => state.process_action(Action::SkipForward),
-            B | PageDown => state.process_action(Action::SkipBack),
-            Z => state.process_action(Action::ToggleFit),
+            W | PageUp => Action::SkipForward,
+            B | PageDown => Action::SkipBack,
+            Z => Action::ToggleFit,
+            Home => Action::First,
```

Comment:
Why are we no longer calling `process_action` on these Actions?

Reply:
Fixed in "Rewrite of Modal system, add Error"

Thanks for catching that, that was completely unintended. This was a mistake in my merging, my bad.